cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,16 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+    cy.get('input[placeholder="Email"]')
+      .type('luckezcy@gmail.com');
+    cy.get('input[placeholder="Password"]')
+      .type('testPassword1!');
```

Comment:
Avoid hardcoding test data directly in your tests. Consider extracting your test data (like email and password) into a separate config or data file. This will make your tests more maintainable and flexible. If you need to change the test data, you will only have to change it in one place.